static/_redirects

```diff
@@ -362,7 +362,7 @@ https://0-20-0.docs.pomerium.com/category/guides https://0-20-0.docs.pomerium.co
 /docs/reference/identity-provider-request-params /docs/reference/identity-provider-settings#identity-provider-request-params
 /docs/reference/identity-provider-scopes /docs/reference/identity-provider-settings#identity-provider-scopes
 /docs/reference/identity-provider-url /docs/reference/identity-provider-settings#identity-provider-url
-/docs/reference/identity-provider-refresh-directory-settings /docs/reference/identity-provider-settings#identity-provider-refresh-directory-settings
+/docs/reference/identity-provider-refresh-directory-settings https://0-21-0.docs.pomerium.com/docs/releases/upgrading#idp-directory-sync
```

Comment:
It looks like this section of the upgrade guide [still exists](https://www.pomerium.com/docs/deploy/core/upgrading#idp-directory-sync); any reason not to link to /docs/deploy/core/upgrading#idp-directory-sync here?

Reply:
Good point. I'll update.